.pre-commit-config.yaml

```diff
@@ -0,0 +1,54 @@
+default_language_version:
+  python: python3
+exclude: '\.(tsv|fasta|gb)$|^ingest/vendored/'
+repos:
+  - repo: https://github.com/snakemake/snakefmt
+    rev: v0.10.1
+    hooks:
+      - id: snakefmt
+        language_version: python3
```

Comment:
I think we need to either bump snakefmt to v0.10.2 or pin python to 3.11 to avoid https://github.com/snakemake/snakefmt/issues/227

Reply:
I set the version to `python3` here because I _don't_ have `python3.11` and having it be that specific was preventing `pre-commit install` from running. 

I will at least up the `snakefmt` version, and then see if that allows a more precise Python pin too.

Reply:
Pushed, thanks for catching!